site/en/guides/border-router/external-commissioning/cli.md

```diff
@@ -72,7 +117,49 @@ device.
 
 ## Join the Thread network
 
-Next, on the Joiner device, [join the Thread network](join.md) and test network connectivity.
+Next, on the Joiner device, [join the Thread network](join.md) and test network
+connectivity.
+
+## Troubleshooting
+
+If you're having issues with OT Commissioner, make sure you use `stop`, then
+`exit` to properly end a session:
+
+```
+> stop
+[done]
+> exit
+```
+
+**IO_ERROR: NET - Reading information from the socket failed**
+
+The socket APIs return this error message when a call to bind or connect to OTBR
+fails. If you're receiving this error message, try the following:
+
+*   Make sure that you're passing the correct port number when you start OT
+    Commissioner. OTBR may use a different port after it's restarted or you
+    reboot your platform.
+*   Make sure that OTBR is running and that your Thread network is properly
+    configured, including your PSKc. Your Passphrase/Commissioner Credential
+    must be a string between 6 and 255 characters.
```

Comment:
Wrong PSKc should result in the next error
```
**SECURITY: SSL - A fatal alert message was received from our peer**
```

Reply:
Done - I reworded this section a bit, and added the new `config get pskc` command.